.github/workflows/golangci-lint.yml

```diff
@@ -18,13 +18,13 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: install Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.18.x
+          go-version-file: go.mod
```

Comment:
Looking at what the action does, I don't think this is going to have any effect _except_ if ruleguard is configured. It should be OK.